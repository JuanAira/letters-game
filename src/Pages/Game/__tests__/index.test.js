import React from 'react';
import { render } from '@testing-library/react';
import { Game } from '..';

describe('<Game /> component', () => {
  const setup = () => {

    const board = ['A', 'B', 'C'];
    const component = render(
        <Game
        board={board}
        />
    );

    return component;
  };

  it('should match snapshot - base', () => {
    const { container } = setup();
    expect(container).toMatchSnapshot();
  });
});
