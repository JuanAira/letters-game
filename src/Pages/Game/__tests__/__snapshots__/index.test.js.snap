// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Game /> component should match snapshot - base 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="section--clear"
    >
      <button
        class="tooltip tooltip--disabled"
        disabled=""
      >
        <svg
          fill="none"
          height="25"
          viewBox="0 0 20 20"
          width="25"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M8.78842 5.03866C8.86656 4.96052 8.97254 4.91663 9.08305 4.91663H11.4164C11.5269 4.91663 11.6329 4.96052 11.711 5.03866C11.7892 5.11681 11.833 5.22279 11.833 5.33329V5.74939H8.66638V5.33329C8.66638 5.22279 8.71028 5.11681 8.78842 5.03866ZM7.16638 5.74939V5.33329C7.16638 4.82496 7.36832 4.33745 7.72776 3.978C8.08721 3.61856 8.57472 3.41663 9.08305 3.41663H11.4164C11.9247 3.41663 12.4122 3.61856 12.7717 3.978C13.1311 4.33745 13.333 4.82496 13.333 5.33329V5.74939H15.5C15.9142 5.74939 16.25 6.08518 16.25 6.49939C16.25 6.9136 15.9142 7.24939 15.5 7.24939H15.0105L14.2492 14.7095C14.2382 15.2023 14.0377 15.6726 13.6883 16.0219C13.3289 16.3814 12.8414 16.5833 12.333 16.5833H8.16638C7.65805 16.5833 7.17054 16.3814 6.81109 16.0219C6.46176 15.6726 6.2612 15.2023 6.25019 14.7095L5.48896 7.24939H5C4.58579 7.24939 4.25 6.9136 4.25 6.49939C4.25 6.08518 4.58579 5.74939 5 5.74939H6.16667H7.16638ZM7.91638 7.24996H12.583H13.5026L12.7536 14.5905C12.751 14.6158 12.7497 14.6412 12.7497 14.6666C12.7497 14.7771 12.7058 14.8831 12.6277 14.9613C12.5495 15.0394 12.4436 15.0833 12.333 15.0833H8.16638C8.05588 15.0833 7.94989 15.0394 7.87175 14.9613C7.79361 14.8831 7.74972 14.7771 7.74972 14.6666C7.74972 14.6412 7.74842 14.6158 7.74584 14.5905L6.99681 7.24996H7.91638Z"
            fill="#939292"
            fill-rule="evenodd"
          />
        </svg>
      </button>
    </div>
    <div
      class="section--board"
    >
      <div
        class="section-board"
      >
        <h3
          class="letter 0 letter--disabled"
        >
          A
        </h3>
        <h3
          class="letter 1 letter--disabled"
        >
          B
        </h3>
        <h3
          class="letter 2 letter--disabled"
        >
          C
        </h3>
        <h3
          class="letter 3 letter--disabled"
        >
          D
        </h3>
        <h3
          class="letter 4 letter--disabled"
        >
          E
        </h3>
        <h3
          class="letter 5 letter--disabled"
        >
          F
        </h3>
        <h3
          class="letter 6 letter--disabled"
        >
          G
        </h3>
        <h3
          class="letter 7 letter--disabled"
        >
          H
        </h3>
        <h3
          class="letter 8 letter--disabled"
        >
          I
        </h3>
        <h3
          class="letter 9 letter--disabled"
        >
          J
        </h3>
        <h3
          class="letter 10 letter--disabled"
        >
          K
        </h3>
        <h3
          class="letter 11 letter--disabled"
        >
          L
        </h3>
        <h3
          class="letter 12 letter--disabled"
        >
          M
        </h3>
        <h3
          class="letter 13 letter--disabled"
        >
          N
        </h3>
        <h3
          class="letter 14 letter--disabled"
        >
          O
        </h3>
        <h3
          class="letter 15 letter--disabled"
        >
          P
        </h3>
      </div>
      <div
        class="section--board section--board--word"
      >
        <h3
          class="word word--invalid"
        >
          <span
            class="word--formed"
          />
          <span
            class="word--feedback"
          >
            invalid
          </span>
        </h3>
      </div>
    </div>
  </div>
</div>
`;
